src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label_big">
```

Comment:
HTML Formatting: The `<label>` element should be indented with 2 spaces inside its parent `<form>` element to maintain proper nesting and readability.